CODEOWNERS

```diff
@@ -1,45 +1,52 @@
-# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

```

Comment:
did the line endings change? I added `?w=1` to see the relevant diff. Just wondering if it will change again in a subsequent PR.

Reply:
I believe it will, even if it won't, we still don't want to have change like this as that makes tracking changes to the file more difficult